app/features/project/project_controller.rb

```diff
@@ -76,9 +76,10 @@ class ProjectController < AuthenticatedControllerBase
     post "#{HOME}/:project_id/save" do
       project_id = params[:project_id]
       project = user_project_with_id(project_id: project_id)
-      project.lane = params["selected_lane"]
+      project.platform = params["selected_lane"].split(" ").first
+      project.lane = params["selected_lane"].split(" ").last
       project.project_name = params["project_name"]
-
+      
```

Comment:
rubocop doesn't complain about trailing white space here?

Reply:
Surely, my local Rubocop integration with vscode is behaving weirdly lately. Gonna check soon!